<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPrincipal</class>
 <widget class="QWidget" name="FormPrincipal">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>910</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>910</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sistema Gráfico Interativo</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>891</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupZoom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1001231</width>
          <height>65</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox  {
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 1ex;
}
 
QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 8px;
}</string>
        </property>
        <property name="title">
         <string>Zoom</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>20</y>
           <width>161</width>
           <height>35</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnZoomOut">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
              <horstretch>30</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>14</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+O</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderControleZoom">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
              <horstretch>70</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnZoomIn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>30</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>14</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>+</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+I</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupNavegacao">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>160</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox  {
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 1ex;
}
 
QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 8px;
}</string>
        </property>
        <property name="title">
         <string>Navegação</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>161</width>
           <height>131</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnNavCenter">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
              <horstretch>35</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>◉</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+Home</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btnNavUp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
              <horstretch>35</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>▲</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+Up</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnNavRight">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
              <horstretch>35</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>▶</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+Right</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btnNavLeft">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
              <horstretch>35</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>◀</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+Left</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btnNavDown">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
              <horstretch>35</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>▼</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+Down</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupRotacao">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox  {
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 1ex;
}
 
QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 8px;
}</string>
        </property>
        <property name="title">
         <string>Rotação</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>19</y>
           <width>161</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QDial" name="dialBtnRotacao">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="sliderPosition">
             <number>1</number>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="wrapping">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupClipping">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox  {
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 1ex;
}
 
QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 8px;
}</string>
        </property>
        <property name="title">
         <string>Clipping de Linhas</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>21</y>
           <width>161</width>
           <height>52</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="radBtnClippingCS">
            <property name="text">
             <string>Cohen-Sutherland</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radBtnClippingLB">
            <property name="text">
             <string>Liang-Barsky</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupProjecao">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox  {
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 1ex;
}
 
QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 8px;
}</string>
        </property>
        <property name="title">
         <string>Tipo de Projeção</string>
        </property>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>21</y>
           <width>161</width>
           <height>52</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="radBtnProjecaoParalela">
            <property name="text">
             <string>Paralela Ortogonal</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radBtnProjecaoPerspectiva">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Perspectiva</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>200</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>480</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="lbObjetos">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Display File</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="displayFile">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::DotLine</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>70</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>17</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>15</number>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Tipo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nome</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnInserirObjeto">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Inserir</string>
          </property>
          <property name="shortcut">
           <string>Alt+I</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnRemoverObjeto">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Remover</string>
          </property>
          <property name="shortcut">
           <string>Alt+R</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnTransformarObjeto">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Transformar</string>
          </property>
          <property name="shortcut">
           <string>Alt+T</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnZoomOut</tabstop>
  <tabstop>sliderControleZoom</tabstop>
  <tabstop>btnZoomIn</tabstop>
  <tabstop>btnNavUp</tabstop>
  <tabstop>btnNavLeft</tabstop>
  <tabstop>btnNavCenter</tabstop>
  <tabstop>btnNavRight</tabstop>
  <tabstop>btnNavDown</tabstop>
  <tabstop>displayFile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
